casemanagementv1/case_management_v1_examples_test.go

```diff
@@ -47,24 +48,25 @@ import (
 // in a configuration file and then:
 // export IBM_CREDENTIALS_FILE=<name of configuration file>
 //
-const externalConfigFile = "../case_management.env"
```

Comment:
To satisfy the linter, I moved all these "global definitions" within examples and int tests into the outer-most Describe() block.  That way there are not conflicts between the examples and int tests for a given service.   These problems only started showing up after I changed the Makefile's "lint" target to include the build tags so that all source files (not just the service and unit tests) are now being lint-checked.

Reply:
Sounds good! Are these changes that will need to be propagated to the generator?

Reply:
> Sounds good! Are these changes that will need to be propagated to the generator?

I opened [this issue](https://github.ibm.com/arf/planning-sdk-squad/issues/3246) to make sure the Go generator is updated as needed.